docs/tutorial/progress-bar.md

```diff
@@ -49,12 +45,25 @@ Starting with a working application from the
 const { BrowserWindow } = require('electron')
 const win = new BrowserWindow()
 
-win.setProgressBar(0.5)
+const INCREMENT = 0.02;
+const INTERVAL_DELAY = 100; //ms
+let c = 0;
+let interval = setInterval(() => {
```

Comment:
Adding a small explanation on what we're doing here with `setInterval` would be helpful (also linking to the node docs: https://nodejs.org/api/timers.html#timers_setinterval_callback_delay_args)

Reply:
@erickzhao is there a way we can embed the files directly so we don't duplicate the code? 
Or because this is never going to show up, is it worth keeping it?

Reply:
Added in commit `dcb5`